doc/configuration.rst

```diff
@@ -492,6 +493,9 @@ missing_data
   For when an :py:class:`~luigi.task.ExternalTask` is not complete, and this
   caused the worker to give up.  As an alternative to fiddling with this, see
   the [worker] keep_alive option.
+complete_failed
+  For signaling that the verification of the completeness of a dependency
+  failed. Typically because some exception have been raised.
```

Comment:
Maybe just `For exceptions in the complete() method`? It doesn't matter if it's a dependency of any task, does it?

Also, how does this relate to "unhandled_exception" which already mentions  ``complete()`` method?